recipes-bsp/imx-vpu-hantro/imx-vpu-hantro_1.18.0.bb

```diff
@@ -2,22 +2,23 @@
 
 DESCRIPTION = "i.MX Hantro VPU library"
 LICENSE = "Proprietary"
-LIC_FILES_CHKSUM = "file://COPYING;md5=fd4b227530cd88a82af6a5982cfb724d"
+LIC_FILES_CHKSUM = "file://COPYING;md5=228c72f2a91452b8a03c4cab30f30ef9"
 
 PROVIDES = "virtual/imxvpu"
 
 SRC_URI = " \
     ${FSL_MIRROR}/${BPN}-${PV}.bin;fsl-eula=true \
 "
-SRC_URI[md5sum] = "79ca586c67ea9b007fa772df8959c852"
-SRC_URI[sha256sum] = "1f373f5d9a37754f7936f9bf4b134b65afe98459c0383b63c50c1f6ecba49e25"
+SRC_URI[md5sum] = "78034de7ed74363eb793d29894bba5e3"
+SRC_URI[sha256sum] = "bebd82649d00d6dd8236b77b8677b1cc6ac46dc474200502df7797a75dc8f568"
```

Comment:
@thochstein here as well, please.

Reply:
It was a bit of work to figure this out, so if you wouldn't mind including the command line and the SHA info, it might help the next time:
```
$ git log --oneline --no-decorate 904a9ab..ee99dc1
ee99dc1 NVJM850-82 - [RV8] RV8 video can't be decoded
b2aa5a0 M865SW-147: [G1/G2]: enhancement for h264 error handing
cfffc5f MMIOT-418: Allow to configure CFG_SECURE_DATA_PATH on Linux
0a741cc NVJM850-81 - [G2]:hevc error handling
2f3f4fa M865SW-132: [G1G2]: HEVC decoder illegal memory access in security mode
8182f16 MA-14108-1 Remove ION_FLAG_CACHED flag for hantro vpu buffer
65d5f3a MMFMWK-8595 Enable VC8000E encoder for 865
b434f22 M865SW-87:[G1/G2]: Integrate some code for VSI platform
4dc2d6f M865SW-82 - [G1/G2]: H264 conformance test failure caused by change in 1.0.6 release
dd606d2 [G1/G2]: Integrate VSI 1.0.7_20191212 release
d62fd74 [G1/G2]: Coverity issues fix
6460a5f [G1/G2]: Refine Test Makefile
7805f7f [G1/G2]: Coverity issues fix
eca7409 [G1/G2]: Coverity issues fix
a210f04 [G1/G2]: Coverity issues fix
dd0e5f4 [G1/G2]: Coverity issues fix
c12e5a8 [G1/G2]: Coverity issues fix
7631926 [G1/G2]: Coverity issues fix
```